lib/output.js

```diff
@@ -95,6 +95,12 @@ module.exports = {
    * @param {CodeceptJS.Step} step
    */
   step(step) {
+    const Locator = require('./locator');
+    if (step.args[1]) {
+      const locator = new Locator(step.args[1]);
```

Comment:
What if 1st argument of a step **is not locator**

Reply:
I think I miss something here, cause the `step.args[0]` or `step.args[1]` could be the locator.